src/main.js

```diff
@@ -39,19 +45,57 @@ function saveIdea(){
   var currentIdea = new Idea(inputTitle.value, inputBody.value);
   list.push(currentIdea);
   displayIdeaCards();
-  console.log(list);
   ideaForm.reset();
   disableEnableButton();
 }
 
+function deleteIdea(){
+  if(event.target.closest(".idea-card")){
+    var target = event.target.closest(".idea-card");
+    ideaCardsGrid.removeChild(target);
+    for(var i = 0; i < list.length; i++){
+      if (target.getAttribute("id") == list[i].id) {
+        list.splice(i , 1);
+      };
+    };
+  };
+};
+
+function toggleFavoriteCard(){
```

Comment:
This is one of the heftier sections I worth reviewing but that I am still excited to see running how we'd like.